Orange/ensembles/ada_boost.py

```diff
@@ -1,24 +1,37 @@
 import sklearn.ensemble as skl_ensemble
-from Orange.base import SklLearner
-from Orange.classification.base_classification import (SklLearnerClassification,
-                                                       SklModelClassification)
-from Orange.regression.base_regression import (SklLearnerRegression,
-                                               SklModelRegression)
 
-__all__ = ["SklAdaBoostLearner", "SklAdaBoostRegressionLearner"]
+from Orange.base import SklLearner, SklModel
+from Orange.classification.base_classification import (
+    SklLearnerClassification, SklModelClassification
+)
+from Orange.modelling import Fitter
+from Orange.regression.base_regression import (
+    SklLearnerRegression, SklModelRegression
+)
+
+__all__ = [
+    'SklAdaBoostLearner',
+    'SklAdaBoosClassificationLearner',
+    'SklAdaBoostRegressionLearner',
+]
 
 
 class SklAdaBoostClassifier(SklModelClassification):
     pass
 
 
-class SklAdaBoostLearner(SklLearnerClassification):
+class SklAdaBoosClassificationLearner(SklLearnerClassification):
     __wraps__ = skl_ensemble.AdaBoostClassifier
     __returns__ = SklAdaBoostClassifier
-    name = 'skl adaBoost'
+    name = 'skl adaBoost classification'
```

Comment:
As far as I see it, it's less redundant this way, so I do prefer it. I don't know whether this plays into @astaric's adored "Implicit is better than explicit" adage, but I find explicit often breaks DRY and is pain to maintain or keep in sync (cf. [_rule of generation_](https://en.wikipedia.org/wiki/Unix_philosophy#Eric_Raymond.E2.80.99s_17_Unix_Rules)), whereas some of the best software that has ever been compiled relies heavily on sane defaults. :smiley: